cpm_data/models.py

```diff
@@ -78,6 +89,11 @@ class Film(ClusterableModel):
         related_name='+'
     )
 
+    objects = SearchableManager()
+
+    def __unicode__(self):
```

Comment:
that's actually a standard way to make human-readable object representations in Python

e397107b2 commit message contains the following:

> Add method __unicode__ to model cpm_data.Film
>
> So it is properly displayed by GenericModelChooserPanel in events app.

do you still want me to add a comment to the code? if yes, I can do it.